connector/routingconnector/config.go

```diff
@@ -39,6 +39,11 @@ type Config struct {
 	// Table contains the routing table for this processor.
 	// Required.
 	Table []RoutingTableItem `mapstructure:"table"`
+
+	// MatchOnce determines whether the connector matches multiple statements.
+	// Valid values are `true` and `false`.
+	// Optional.
+	MatchOnce bool `mapstructure:"match_once"`
```

Comment:
Can you add the new config options to the README? Please include defaults (if applicable), usage description, and config examples.

Reply:
Sure, I have added the new config options to the README. Please take a look and let me know if everything is clear and meets your expectations. Thank you!